Package.swift

```diff
@@ -28,7 +28,7 @@ let package = Package(
         )
     ],
     dependencies: [
-        .package(url: "https://github.com/Kitura/BlueRSA.git", from: "1.0.31"),
+        .package(url: "https://github.com/Kitura/BlueRSA.git", from: "1.0.200"),
         .package(url: "https://github.com/Kitura/BlueCryptor.git", from: "1.0.0"),
         .package(url: "https://github.com/Kitura/BlueECC.git", from: "1.1.0"),
         .package(url: "https://github.com/Kitura/LoggerAPI.git", from: "1.7.0"),
```

Comment:
The logger API also needs to be updated before merging this PR, with a new tag.

Reply:
Thanks @mbarnach I'm aware.  I'm just through each dependency and re-triggering CI here to confirm.  Sorry for the noise.